tests/visualization_tests/test_timeline.py

```diff
@@ -148,6 +178,21 @@ def test_get_timeline_plot(
         plt.close()
 
 
+@parametrize_plot_timeline
+@pytest.mark.parametrize(
+    "n_recent_trials",
+    [0, -1, -10],
+)
+def test_plot_timeline_n_recent_trials_invalid(
+    plot_timeline: Callable[..., Any],
+    n_recent_trials: int | None,
+) -> None:
+    states = [TrialState.COMPLETE, TrialState.PRUNED, TrialState.FAIL, TrialState.RUNNING]
+    study = _create_study(states)
+    with pytest.raises(ValueError, match="n_recent_trials must be a positive integer or None"):
```

Comment:
Checking error messages can be fragile, so let me remove the `match` argument in a follow-up PR.